resources/assets/js/jsx/profile_page.jsx

```diff
@@ -53,16 +53,23 @@
 					dataType: 'json'
 			})
 			.done(function(userData) {
+				this.props.user.cover.id = userData.data.cover.id;
```

Comment:
Yes but for some reason their was an error where when a user selects a cover, then re-enters a different box the selected cover will go back to its previous one. This solved that for me.